api/src/main/java/org/openmrs/PatientIdentifier.java

```diff
@@ -328,4 +331,23 @@ public int compare(PatientIdentifier pi1, PatientIdentifier pi2) {
 			return retValue;
 		}
 	}
+
+
+	/**
+	 * Gets patient program associated to the identifier in context
+	 * @since 2.3.6
+	 * @return patientProgram the patient program associated to an identifier
+	 */
+	public PatientProgram getPatientProgram() {
+		return patientProgram;
+	}
+
+	/**
+	 * This method sets the patient program on a patient Identifier
+	 * @since 2.3.6
+	 * @param patientProgram The patientProgram to set.
+	 */
+	public void setPatientProgram(PatientProgram patientProgram) {
```

Comment:
This would also be good to be like  @param patientProgram The patient program to set.